packages/authentication/ios/Plugin/FirebaseAuthenticationPlugin.swift

```diff
@@ -570,14 +570,8 @@ public class FirebaseAuthenticationPlugin: CAPPlugin, CAPBridgedPlugin {
             return
         }
 
-        guard let actionCodeSettingsDict = call.getObject("actionCodeSettings") else {
-            call.reject(errorActionCodeSettingsMissing)
-            return
-        }
-        guard let actionCodeSettings = FirebaseAuthenticationHelper.createActionCodeSettings(actionCodeSettingsDict) else {
-            call.reject(errorActionCodeSettingsMissing)
-            return
-        }
+        let actionCodeSettingsDict = call.getObject("actionCodeSettings")
+        let actionCodeSettings = actionCodeSettingsDict != nil ? FirebaseAuthenticationHelper.createActionCodeSettings(actionCodeSettingsDict!) : nil
```

Comment:
```suggestion
        let actionCodeSettings: ActionCodeSettings?
        if let actionCodeSettingsDict = call.getObject("actionCodeSettings") {
        	actionCodeSettings = FirebaseAuthenticationHelper.createActionCodeSettings(actionCodeSettingsDict)
        }
```

We try not to force types.